tests/plugins/easyimage/customstyleicons.js

```diff
@@ -0,0 +1,73 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: floatingspace,easyimage,toolbar */
+
+( function() {
+	'use strict';
+
+	var buttonCreated = false,
+		commonConfig = {
+			easyimage_styles: {
+				test: {
+					icon: 'tests/_assets/sample_icon.png',
+					iconHiDpi: 'tests/_assets/sample_icon.hidpi.png'
+				}
+			},
+			easyimage_toolbar: [ 'EasyimageTest' ]
+		};
+
+	bender.test( {
+		setUp: function() {
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) {
+				assert.ignore();
+			}
+
+			this.addButtonStub = sinon.stub( CKEDITOR.ui.prototype, 'addButton', function( name, definition ) {
```

Comment:
Button stub causes tests to fail on built version of the editor due to the fact that `contextmenu` plugin is loaded and `addContextMenuItems` uses `editor.ui.items` to get info from toolbar's buttons.

Probably spying instead of stubbing will fix the issue.

Reply:
`Spy` doesn't give an easy solution to do this assert same way as with stub (or I am just bad at searching). I just added call to original function in the stub so it does not affect a normal flow in any way.